atat/routes/dev.py

```diff
@@ -183,23 +188,36 @@ def get_or_create_dev_persona(persona):
 
 @local_access_bp.route("/dev-new-user")
 def dev_new_user():
-    first_name = request.args.get("first_name", None)
-    last_name = request.args.get("last_name", None)
-    dod_id = request.args.get("dod_id", None)
+    first_name = request.args.get("first_name", None, type=str)
+    last_name = request.args.get("last_name", None, type=str)
+    dod_id = request.args.get("dod_id", None, type=str)
```

Comment:
Unless I am mistaken, URL query parameters are always going to be strings. Further, `None` is the default value for the `default` parameter. So the equivalent code is
```suggestion
    first_name = request.args.get("first_name")
    last_name = request.args.get("last_name")
    dod_id = request.args.get("dod_id")
```